zkEVM smart contract address & RPC update (#1139)
maticnetwork/matic-docs: docs/zkEVM/develop.md

Developers will be able to deploy their existing contracts to the zkEVM, and Users can deposit assets from Ethereum and transact off-chain. These transactions are grouped into batches with zero-knowledge proof attesting to the validity of each transaction.

## **Connecting to zkEVM**

> In order to add the **Polygon zkEVM** network to your wallet, you will require the following details:

Reviewer: ```suggestion
In order to add the **Polygon zkEVM** network to your wallet, you will need to enter the following details:
```

- **Network Name**: Polygon zkEVM Testnet
- **RPC URL**: [https://public.zkevm-test.net:2083](https://public.zkevm-test.net:2083)
- **Chain ID**: 1402
- **Currency Symbol**: ETH
- **Block Explorer URL**: [https://public.zkevm-test.net:2083](https://public.zkevm-test.net:2083)

> Also, for moving assets across chains, you will require the zkEVM Bridge. It is available on [https://public.zkevm-test.net/](https://public.zkevm-test.net/).

Reviewer: ```suggestion
Also, for moving assets across chains, you will need to use the zkEVM Bridge. The bridge is available on [https://public.zkevm-test.net/](https://public.zkevm-test.net/).
```
